README.md

```diff
@@ -10,8 +10,14 @@ Sticky and Collapsible View Controller on top of tab bar
 
 ## Installation
 
-StickyTabBarViewController is available through [CocoaPods](http://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+StickyTabBarViewController is available through [SPM](https://swift.org/package-manager/) and [CocoaPods](http://cocoapods.org).
```

Comment:
Totally a nitpick but it'd be nice to have the name of the library back-ticked.

Reply:
In a way that it would be included in the following dotted points or just have a tick visible before the name? So like:

- Sticky...

or 

. Sticky (like other points and indented)

@Thurman1776-

Reply:
I meant something like this: 

- `StickyTabBarViewController` is available through...  

But is really no big deal, you can just merge as is 👍

Reply:
Thank you! @Thurman1776-